tests/test_views.py

```diff
@@ -0,0 +1,74 @@
+# Packages
+from unittest import TestCase
+
+# Local
+from webapp.views import shorten_acquisition_url
+
+
+class TestViewsFunctions(TestCase):
```

Comment:
I'd maybe rename this to something like `TestMarketoFunctions`? `ViewFunctions` is quite generic, right? It could be any Flask View function

Reply:
Oh I see, was it your purpose to create a new set of view tests that everyone can use in the future?

Reply:
Yh, I created the file so other view tests could be written in there as well.